api/migrations/versions/36235e184d09_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
```

Comment:
Strange... it seems like the migration didn't pick up your new model! I suspect it might be because it can't trace a reference to it... Can you try import your model in [__init__.py](https://github.com/deep-learning-indaba/Baobab/blob/develop/api/app/__init__.py), then create a new migration?